app/models/manageiq/providers/openstack/inventory/persister.rb

```diff
@@ -21,6 +24,13 @@ def initialize(manager, target = nil, collector = nil)
 
   protected
 
+  # TODO: this reads whole table ContainerLabelTagMapping.all.
+  #   Is this expensive for each targeted refresh?
```

Comment:
Good question, maybe @Ladas or @agrare might know?  Otherwise I fine with merging.

Reply:
That's a comment copied from @Ladas' original code ;)

Reply:
Ah, copied from merged code :-)